### PR TITLE
 #112 Added support to apply 'opens' directive on a resource-only package, a mandatory feature for JavaFX applications

### PR DESCRIPTION
**Pull request by xinyingho**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.moditect</groupId>
         <artifactId>moditect-parent</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <version>1.0.0-negatron</version>
```
Comment (gunnarmorling) [inline core/pom.xml]: This shouldn't change (same in a few other POMs).

File: core/src/main/java/org/moditect/commands/GenerateModuleInfo.java [hunk 2]
```diff
@@ -232,11 +247,32 @@ private void updateModuleInfo(Map<String, Boolean> optionalityPerModule, ModuleD
         if ( moduleName != null ) {
             moduleDeclaration.setName( moduleName );
         }
+        
+        opensResources.stream().forEach(resourcePackage -> moduleDeclaration.getDirectives().add(
```
Comment (gunnarmorling) [inline core/src/main/java/org/moditect/commands/GenerateModuleInfo.java]: I see no mention of this in the README. Could you add it?

File: core/src/main/java/org/moditect/commands/GenerateModuleInfo.java [hunk 3]
```diff
@@ -345,6 +381,9 @@ private ModuleDeclaration applyOpensPatterns(ModuleDeclaration moduleDeclaration
 
     private ModuleDeclaration parseGeneratedModuleInfo() {
         Path moduleDir = workingDirectory.resolve( autoModuleNameForInputJar );
+        int multiReleaseIdx = jdepsExtraArgs.indexOf("--multi-release");
```
Comment (gunnarmorling) [inline core/src/main/java/org/moditect/commands/GenerateModuleInfo.java]: Does this need mention in the README?

File: core/pom.xml (hunk 1 above)
Comment (xinyingho) [reply core/pom.xml]: I thought that you abandonned your project and I was preparing to go solo with my own fork :) It seems like I'm wrong so I'll rollback this particular change.

File: core/src/main/java/org/moditect/commands/GenerateModuleInfo.java (hunk 3 above)
Comment (xinyingho) [reply core/src/main/java/org/moditect/commands/GenerateModuleInfo.java]: No, it's not needed but this particular modification is currently set to work only with jdeps 14+, it's not compatible with previous versions anymore. I'll modify it to make it compatible with jdeps 9-13.

File: core/src/main/java/org/moditect/commands/GenerateModuleInfo.java (hunk 2 above)
Comment (xinyingho) [reply core/src/main/java/org/moditect/commands/GenerateModuleInfo.java]: done

File: core/src/main/java/org/moditect/commands/GenerateModuleInfo.java (hunk 3 above)
Comment (xinyingho) [reply core/src/main/java/org/moditect/commands/GenerateModuleInfo.java]: done as well

File: core/pom.xml (hunk 1 above)
Comment (xinyingho) [reply core/pom.xml]: done

**Comment by gunnarmorling** [comment]
Applied; thanks, @xinyingho!

**Pull request merged**